rules/docker-nat.mk

```diff
@@ -31,4 +31,4 @@ $(DOCKER_NAT)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
 $(DOCKER_NAT)_RUN_OPT += -v /host/warmboot:/var/warmboot
 
 $(DOCKER_NAT)_BASE_IMAGE_FILES += natctl:/usr/bin/natctl
-
+$(DOCKER_NAT)_BASE_IMAGE_FILES += $(SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT)
```

Comment:
Check builds are failing because `SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT` should be added to `$(DOCKER_NAT)_FILES`, not `$(DOCKER_NAT)_BASE_IMAGE_FILES`

Reply:
Please help me review again!